### DISCUSSION TITLE
ImportError: numpy.core.multiarray failed to import

### DISCUSSION THREAD

**Discussion by efdalkaya** [UMEP-dev/UMEP#518] [Python]
Hello everyone,

How can fix the problem?

![1](https://user-images.githubusercontent.com/17739723/236429138-d9a13c1a-16ef-4b4b-a72e-bd3fafe1e6a4.png)

**Comment by biglimp** [comment]
I suspect you have an older version of numpy or some other version conflict. See if this link can be of any help: https://umep-docs.readthedocs.io/en/latest/Getting_Started.html#adding-missing-python-libraries-and-other-osgeo-functionalities

**Reply by jency-phj** [reply]
I also encountered the same problem, but I didn't solve it. How did you solve it later?

**Comment by jency-phj** [comment]
I also encountered the same problem, but I didn't solve it. How did you solve it later?
